compiler-builtins/src/avr.rs

```diff
@@ -0,0 +1,22 @@
+intrinsics! {
+    pub unsafe extern "C" fn abort() -> ! {
+        // On AVRs, an architecture that doesn't support traps, unreachable code
+        // paths get lowered into calls to `abort`:
+        //
+        // https://github.com/llvm/llvm-project/blob/cbe8f3ad7621e402b050e768f400ff0d19c3aedd/llvm/lib/CodeGen/SelectionDAG/LegalizeDAG.cpp#L4462
+        //
+        // When control gets here, it means an undefined behavior has occurred, so
```

Comment:
Not necessarily undefined behavior. `core::intrinsics::abort()` also lowers to a call to `abort` on AVR.

Reply:
true, true!